aws/data_source_aws_eip_test.go

```diff
@@ -219,10 +219,6 @@ data "aws_eip" "test" {
 `
 
 const testAccDataSourceAwsEipConfigPublicIpEc2Classic = `
-provider "aws" {
```

Comment:
Since its not a subject that will be dealt with with any future testing, will migrate this to the standardized EC2-Classic handling (https://github.com/hashicorp/terraform-provider-aws/pull/15803) post-merge. It was missed before because it wasn't using `os.Setenv()` like the others.